src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
-                    {
+                    if (Group* group = IsPet() ? GetOwner()->ToPlayer()->GetGroup() : ToPlayer()->GetGroup())
                         for (GroupReference* itr = group->GetFirstMember(); itr != NULL; itr = itr->next())
-                        {
```

Comment:
Code style says brackets should be used whenever there is more than one line in the code block.